### PR TITLE
Use Stateful object to check if Interactive Login have been triggered

### PR DESCRIPTION
**Pull request by henrikbjorn**
HHVM behaves differently when passing values as references. This removes
the need of doing that.

### CODE REVIEW

File: tests/Silex/Tests/Provider/RememberMeServiceProviderTest.php [hunk 1]
```diff
@@ -96,3 +95,12 @@ public function createApplication($authenticationMethod = 'form')
         return $app;
     }
 }
+
+class InteractiveLoginTriggered {
```
Comment (fabpot) [inline tests/Silex/Tests/Provider/RememberMeServiceProviderTest.php]: `{` on the next line.

**Comment by henrikbjorn** [comment]
Fixed the small code style issue.

**Comment by fabpot** [comment]
Thank you @henrikbjorn.

**Comment by fabpot** [comment]
I didn't see this was done on master. I've cherry-picked the commit on 1.3.

**Comment by stof** [comment]
@henrikbjorn have you reported the PHP incompatibility to the HHVM team here ?

**Comment by henrikbjorn** [comment]
@stof nope, just assumed it was correct, also this is proberly the better way of testing this.

**Pull request closed without merging**